tools/package_tools/mmdeploy_builder.py

```diff
@@ -283,42 +317,63 @@ def create_package(cfg: Dict, mmdeploy_dir: str):
         dist_dir = osp.join(build_dir, 'dist')
         build_mmdeploy(cfg, mmdeploy_dir, dist_dir=dist_dir)
 
-        if build_sdk_flag:
+        if build_sdk_flag == 'ON':
 
             sdk_tar_dir = osp.join(build_dir, sdk_tar_name)
 
             # copy lib and install into sdk dir
             install_dir = osp.join(mmdeploy_dir, 'build/install/')
             _copy(install_dir, sdk_tar_dir)
+            _copy(f'{mmdeploy_dir}/demo/python',
+                  f'{sdk_tar_dir}/example/python')
             _remove_if_exist(osp.join(sdk_tar_dir, 'example', 'build'))
 
-            # create sdk python api wheel
-            # for linux
-            python_api_lib_path = glob(
-                osp.join(mmdeploy_dir, 'build/lib/mmdeploy_python.*.so'))
-            # for windows
-            python_api_lib_path += glob(
-                osp.join(mmdeploy_dir, 'build/bin/*/mmdeploy_python.*.pyd'))
-            num_libs = len(python_api_lib_path)
-            if num_libs != 1:
-                logging.info('find multiple mmdeploy_python libraries.')
-            python_api_lib_path = python_api_lib_path[0]
-
-            sdk_python_package_dir = osp.join(build_dir, '.mmdeploy_python')
-            _copy(PACKAGING_DIR, sdk_python_package_dir)
-            _copy(
-                osp.join(mmdeploy_dir, 'mmdeploy', 'version.py'),
-                osp.join(sdk_python_package_dir, 'mmdeploy_python',
-                         'version.py'))
-            _copy(python_api_lib_path,
-                  osp.join(sdk_python_package_dir, 'mmdeploy_python'))
-            sdk_wheel_dir = osp.abspath(osp.join(sdk_tar_dir, 'python'))
-            bdist_cmd = _create_bdist_cmd(
-                cfg, c_ext=True, dist_dir=sdk_wheel_dir)
-            _call_command(bdist_cmd, sdk_python_package_dir)
-
-            # remove temp package dir
-            _remove_if_exist(sdk_python_package_dir)
+            # build SDK Python API according to different python version
+            # 1. get conda env path from python's path.
+            # We assume python is managed by conda
```

Comment:
Will we support building on non-conda env in the future?

Reply:
I am not sure. In my opinion, it depends on specific situations. 
Currently, we only support precompiling on windows-x64 and linux-x86_64 platforms. So, I think conda is fair enough.